src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -750,7 +756,9 @@ private void DetectAttributeAndOptionConflicts(DiagnosticBag diagnostics)
 
             ThreeState assemblyDelaySignAttributeSetting = this.AssemblyDelaySignAttributeSetting;
             if (_compilation.Options.DelaySign.HasValue && (assemblyDelaySignAttributeSetting != ThreeState.Unknown) &&
+#nullable disable // Should this use '_compilation' instead of 'DeclaringCompilation'? Can 'DeclaringCompilation' be null here? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
Not sure why the question here? `DeclaringCompilation` just returns `_compilation`, and neither are annotated? Should one of them be?

Reply:
I'm planning to leave this for a follow-up